frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalar.jsx

```diff
@@ -60,6 +61,20 @@ export class SmartScalar extends Component {
   static noHeader = true;
 
   static settings = {
+    ...fieldSetting("scalar.field", {
+      title: t`Field to show`,
+      fieldFilter: isNumeric,
+      getDefault: ([
+        {
+          data: { cols },
+        },
+      ]) => cols[0].name,
```

Comment:
Based on the line `const metricIndex = cols.findIndex(col => !isDate(col));` of the previous version of `SmartScalar` the default column should be `cols.findIndex(col => !isDate(col)).name` otherwise it will break all existing Trend charts

Reply:
3d259d4850ebf6f01bb46744a2bc621ab0dac33c